### ISSUE TITLE
Einträge filtern nach Namespace

### ISSUE THREAD

**Issue by cukabeka** [FriendsOfREDAXO/feeds#41]
Wenn ich viele Quellen habe, ist nützlich, auch im Backend die Einträge filtern zu können.

**Comment by alexplusde** [comment]
wäre gelöst mit #74

**Comment by skerbis** [comment]
Es wurde nun eine Sortierung hinzugefügt. Gibt es irgendwo ein Beispiel für einen Filter mit rex list?

**Comment by alxndr-w** [comment]
Nein. Deswegen war mein Vorschlag auch, voll auf YForm zu setzen und dessen UI - z.B. Filtermöglichkeiten und Suche - zu nutzen, statt was eigenes zu bauen.

**Comment by skerbis** [comment]
Dann werd‘ ich einen realisieren

**Comment by aeberhard** [comment]
@skerbis 
![image](https://user-images.githubusercontent.com/330560/212179929-18fbfde6-37e1-4738-9947-554cb56184f3.png)

Meinst Du ein Suchfeld? Das habe ich grade im Consent-Manmager eingebaut. Neue Beta kommt heute noch raus.
Das ist auch eine REX-LIST. Im Core wurden dafür auch Anpasssungen gemacht um dieses Suchfeld auch in anderen Addons verwenden zu können.

**Comment by skerbis** [comment]
Nein. Einfach nur n Select mit Kategorie-Auswahl oder Key ... 
Eine Suche brauche ich nicht, da Feeds ja Watson unterstützt.

**Comment by skerbis** [comment]
erledigt
